app/routes/app/$mType/m/$laborMarketAddress/prereqs.tsx

```diff
@@ -12,6 +12,13 @@ export default function MarketplaceIdPrerequesites() {
     throw new Error("MarketplaceIdPrerequesites must be rendered under a MarketplaceId route");
   }
   const { laborMarket } = data;
+  if (!laborMarket) {
```

Comment:
can use `invariant` here to one line this

Reply:
Gives errors about labor market being possibly null

Reply:
nvm